bokeh/models/renderers.py

```diff
@@ -45,6 +48,28 @@ def _check_bad_column_name(self):
         if missing:
             return "%s [renderer: %s]" % (", ".join(sorted(missing)), self)
 
+    @validation.warning(COLON_IN_CATEGORY_LABEL)
+    def _check_colon_in_category_label(self):
+        if not self.glyph: return
+        if not self.data_source: return
+        vm = self.glyph.vm_serialize()
+        labels = (label for label in ['x', 'y']
+                  if label in vm and 'field' in vm[label])
```

Comment:
@bryevdv looking at https://github.com/bokeh/bokeh/blob/master/examples/glyphs/glyphs.py I see all different kind of ways passing in values.
1. `MultiLine` would require separate check, because data source is list of lists
2. Is there currently a way to get available axis (or how do you call `x`, `xs` etc)?
3. Not sure if hard coding all   `x`, `y`, `xs`, `ys`, `xp02` etc, is worthwhile